src/pages/Home/home.js

```diff
@@ -1,7 +1,156 @@
-export const Home = () => {
-  return (
-    <>
-      New Home
-    </>
-  )
-}
\ No newline at end of file
+import axios from "axios";
+import React from "react";
+import { Link } from "react-router-dom";
+import { Container } from "./home.styles";
+import { Card } from "./home.styles";
+
+export class Home extends React.Component {
+  state = {
+    horror: {},
+    romance: {},
+    mystery: {},
+    nonfiction: {},
+    history: {},
+  };
+
+  getBookData = async (subject) => {
+    try {
+      const { data } = await axios(
+        `https://www.googleapis.com/books/v1/volumes?q=subject:${subject}&maxResults=40&key=${process.env.REACT_APP_API_KEY}`
+      );
+      if (subject === "horror") {
```

Comment:
I'd maybe change this for a switch statement, looks cleaner and you avoid writing so many 'subject ==='

Reply:
Solid advice...Cristian showed me how to clean it up even more in the call.